lib/resin.coffee

```diff
@@ -251,7 +251,7 @@ getSdk = (opts = {}) ->
 # });
 ###
 getSdk.setSharedOptions = (opts) ->
-	root = if window? then window else GLOBAL
+	root = if window? then window else global
```

Comment:
How about creating a single global object provider?
Also, in the past I've used:
```
typeof window !== 'undefined' ? window :
    typeof self !== 'undefined' ? self :
    global
```
so that it also works inside workers. It might be an overkill here though.

Reply:
Good idea! It would definitely be nice to try and get this working in web workers too. I've updated it to use almost exactly that logic.